src/AutoRest.CSharp/LowLevel/Output/DpgOutputLibraryBuilder.cs

```diff
@@ -385,7 +411,22 @@ private class ClientInfo
             public string Description { get; }
             public INamedTypeSymbol? ExistingType { get; }
             public IReadOnlyList<InputOperation> Operations { get; }
-            public IReadOnlyList<InputParameter> ClientParameters { get; }
+
+            private IReadOnlyList<InputParameter> _clientParameters;
+            public IReadOnlyList<InputParameter> ClientParameters
+            {
+                get
+                {
+                    if (_clientParameters != null && _clientParameters.Count > 0)
```

Comment:
You did not declare the variable `_clientParameters` as nullable, we do not need this check when it is not nullable

Reply:
resolved